_pages/saml/authentication.md

```diff
@@ -114,8 +171,18 @@ A proofed identity request at AAL2, with phishing resistent MFA, for email, phon
 <div class="grid-row grid-gap">
     <div class="desktop:grid-col-7 mobile:grid-col-full">
         <h2 id="authentication-response">Authentication response</h2>
-        <p>After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL:</p>
-        <p>The SAMLResponse is a base64-encoded XML payload that contains encrypted data.</p>
+        <p markdown="1">After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL with a hidden form control named `SAMLResponse`.</p>
+        <p markdown="1">`SAMLResponse` contains a base64-encoded XML payload that contains data that is encrypted with the service provider's public key.</p>
+        <p markdown="1"> The decrypted `SAMLResponse` contains a `<saml:Assertion>` element, which in turn contains the following elements: </p>
+        <dl>
+            <dt markdown="1">`Subject`</dt>
+            <dd>Contains the NameID, the Recipient of this information and the validity period.</dd>
+            <dt markdown="1">`AttributeStatement`</dt>
+            <dd>All the requested attributes.</dd>
+            <dt markdown="1">`AuthnStatement`</dt>
+            <dd>Contains the AAL that was used.</dd>
+        </dl>
+        <p>For example, {{ decrypted_response | markdownify }}</p>
```

Comment:
```suggestion
        <p>For example: {{ decrypted_response | markdownify }}</p>
```

Reply:
i am wondering if it is more useful to have this example in the `example` tab than what is currently there. (although to be honest i can't see someone spending a lot of time looking at either, since as noted above by aj, it's pretty unreadable in its current state)

Reply:
I thought about that and felt that it was also useful to retain an example of the encrypted response.

Reply:
i think maybe it would make more sense than as adding another tab to the thing on the right - it doesn't really make sense to have an XML example in the body and in the tab. we could have `Encrypted Response` and `Decrypted Response`.

Reply:
Unfortunately, that's a bigger lift. There's a lot of JavaScript that assumes that the "code" section has only 2 tabs.

Reply:
then i think we should pause on this -- it'll need a UX/content review, and i hesitate to add that to ursula's plate.